examples/MomentoRedisLambdaExample/MomentoRedisExampleLambdaHandler/src/MomentoRedisExampleLambdaHandler/RedisCache.cs

```diff
@@ -0,0 +1,143 @@
+using System.Diagnostics;
+using StackExchange.Redis;
+
+namespace MomentoRedisExampleLambdaHandler;
+
+public class RedisCache
+{
+    private IDatabase _db;
+    public RedisCache(ConnectionMultiplexer redis)
+    {
+        _db = redis.GetDatabase();
+    }
+    public void PingRedis() {
+        // Check/Ping the redis server
+        Console.WriteLine("Pinging db");
+        var pong = _db.Ping();
+        Console.WriteLine(pong);
+    }
+
+    public async Task<List<Task<RedisSetResult>>> DoRedisSets(int numRedisSetRequests)
+    {
+        Console.WriteLine("Executing Redis Sets");
+        try
+        {
+            List<Task<RedisSetResult>> redisSetTasks = new List<Task<RedisSetResult>>();
+            string sourceData = await Utils.GetSourceData();
```

Comment:
same comment here about making this function sync and passing this sourcedata in as an arg